### PR TITLE
Upload version name and collector data

### PR DESCRIPTION
**Pull request by martin-ngyn**
## 💬 Summary of Changes

Send version number and collector name through the RunEnvironment when uploading test analytics.

### CODE REVIEW

File: collector/test-data-uploader/src/main/kotlin/com/buildkite/test/collector/android/models/RunEnvironment.kt [hunk 1]
```diff
@@ -38,4 +42,11 @@ data class RunEnvironment(
         return ciRunEnvironment
             ?: localRunEnvironment
     }
+
+    companion object {
+        // When bumping version, update VERSION_NAME to match new version
+        // Used for uploading correct library version
+        val VERSION_NAME = "0.2.0-SNAPSHOT"
+        val COLLECTOR_NAME = "test-collector-android"
```
Comment (KatieWright26) [inline collector/test-data-uploader/src/main/kotlin/com/buildkite/test/collector/android/models/RunEnvironment.kt]: Hi @martin-ngyn, I did recently did some work on our other collectors with adding a collector name; the syntax is `{language}-buildkite-test-collector`. Could this be changed to be "android-buildkite-test-collector" to match?

Comment (martin-ngyn) [reply collector/test-data-uploader/src/main/kotlin/com/buildkite/test/collector/android/models/RunEnvironment.kt]: Done 🙂

**Review by KatieWright26 — COMMENTED** [review]
Thanks for making that change @martin-ngyn! I have one other request, which was could you please add to any existing test for the RunEnvironment to make sure the version and collector name are definitely coming through as expected?

Once this is merged, will y'all arrange the release?

**Review by KatieWright26 — APPROVED** [review]
Thanks @martin-ngyn! I don't have the means to test this locally, but this LGTM :)

**Comment by thebhumilsoni** [comment]
LGTM.

**Pull request merged**